neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -44,7 +71,7 @@ class NeuralynxRawIO(BaseRawIO):
             Display all information about signal channels, units, segment size....
     """
     extensions = ['nse', 'ncs', 'nev', 'ntt']
-    rawmode = 'one-dir'
+    rawmode = 'one-dir'  # :TODO: remove this line as now set in __init__
```

Comment:
No. This must be a class attribute because it is used for some GUI to open dialoig as "file" or "dir" mode.
Must not be an instance attribute.
Leave it in class attribute.